syntax/decode_test.go

```diff
@@ -153,6 +146,46 @@ func TestDecodeSlice(t *testing.T) {
 	}
 }
 
+func TestDecodeSliceZeroHead(t *testing.T) {
+	type inner struct {
+		X []byte
+	}
+
+	encoded := bytes.Repeat([]byte{0xA0}, 64)
```

Comment:
Please follow the `x/y/z` convention that the other test cases follow.  Suggest `xv0` and `xvV`.